instance/config.py

```diff
@@ -1,15 +1,26 @@
-class Config:
+import os
+
+class Config(object):
     DEBUG = False
     TESTING = False
-
+    SECRET_KEY = os.getenv('SECRET_KEY')
 
 class Development(Config):
     DEBUG = True
 
-
 class Testing(Config):
     TESTING = True
+    DEBUG = True
 
+class Staging(Config):
```

Comment:
I don't think you will be using staging environment so you can just do away with it